endpoints/10-cluster-endpoints.md

```diff
@@ -0,0 +1,319 @@
+
+Note that `nodeName` should be unique name for each new worker node to be added. It must follow naming conventions (no spaces, special characters, or uppercase letters), as it will be used as part of the cluster node identifier.
+
+### 10.7- ScaleIn endpoint:
+
+**Description**:
+This endpoint allows users to remove specific worker nodes from a Kubernetes cluster. This operation is essential for efficiently managing cluster resources, especially when scaling down applications or reconfiguring the cluster architecture. By invoking this endpoint, you can effectively decommission nodes that are no longer needed or are underutilized.
+
+Before invoking the ScaleIn endpoint, ensure that the workloads on `worker_name` have been migrated to other nodes, by using the LabelNodes endpoint to mark it as unavailable and using ManageApplication endpoint to reduce the number of the application replicas.
+
+After the ScaleIn operation, the removal of worker nodes and their status can be tracked using the GetCluster endpoint or monitored in the ProActive. The created workflows correspond to the one which is used to remove nodes during DeleteCluster operation.,
+
+**Path:**
+
+```url
+🟡 POST {{protocol}}://{{sal_host}}:{{sal_port}}/sal/cluster/{{cluster_name}}/scalein
+```
+
+**Headers:** `sessionid`
+
+**Body:** Json input following this format:
+
+```json
+[
+  "{{worker_name}}",
+  "{{worker_name2}}"
+]
+```
+
+**Reply:** JSON format represented with the [ClusterNodeDefinition](https://github.com/ow2-proactive/scheduling-abstraction-layer/blob/master/sal-common/src/main/java/org/ow2/proactive/sal/model/ClusterNodeDefinition.java) and status values corresponding to ones observed in ProActive dashboard.
+
+Note that the worker names should correspond to existing ones in the cluster, otherwise the operation will be rejected.
+
+### 10.8- LabelNode endpoint:
+
+**Description**: This endpoint allows users to manage node labels within a Kubernetes cluster by adding, modifying, or removing labels. Labels are key-value pairs that categorize nodes, making it easier to target specific nodes for application deployment, scaling, or management tasks.
+Using LabelNodes, you can dynamically adjust labels on worker nodes as you manage scaling operations. During ScaleOut, nodes are labeled to mark them as available for application deployments. Before invoking ScaleIn, labels can be updated to indicate that certain nodes should no longer be scheduled for application workloads, preparing them for safe removal.
+
+**Path:**
+
+```url
+🟡 POST {{protocol}}://{{sal_host}}:{{sal_port}}/sal/cluster/{{cluster_name}}/label
+```
+
+**Headers:** `sessionid`
+
+**Body:** Json input following this format:
+
+```json
+[
+  // to add lable it is to use value 'yes' and to remove 'no'
```

Comment:
not sure the comment should be put inside the json format, as json does not support comments

I would rather write:
`
**Body:** Json input following this format ('yes' means a label will be added and 'no' means a label will be removed):

`